src/plugins/simulator/simulations/terrarium/terrarium.tsx

```diff
@@ -20,7 +31,89 @@ const fanTemperatureImpactPerStep = -1 / 60000 * stepDuration; // -1 degree/minu
 const heatLampTemperatureImpactPerStep = 1 / 60000 * stepDuration; // +1 degree/minute
 const humidifierHumidityImpactPerStep = 15 / 60000 * stepDuration; // +15%/minute
 
+function TerrariumComponent({ frame, variables }: ISimulationProps) {
+  const humidifierFrameRef = useRef(0);
+  const geckoFrameRef = useRef(0);
+  const geckoDirectionRef = useRef(1);
+
+  // Determine temperature reading
+  const temperatureVariable = findVariable(kTemperatureKey, variables);
+  const temperatureValue = temperatureVariable?.currentValue ?? 0;
+  const temperatureReading = `${Math.round(temperatureValue)}°C`;
```

Comment:
Not a big deal, but I think of a "reading" as the raw value from the sensor.  So since this is a string that includes the units interpolated in, and no longer has value to us as a "reading",  could we change this variable name to `temperatureDisplay`? and likewise `humidityReading` to `humidityDisplay`?